core/src/main/java/nu/yona/server/device/entities/UserDevice.java

```diff
@@ -16,18 +16,31 @@
 
 import org.hibernate.annotations.Type;
 
+import nu.yona.server.crypto.CryptoUtil;
 import nu.yona.server.crypto.seckey.DateFieldEncryptor;
 import nu.yona.server.crypto.seckey.DateTimeFieldEncryptor;
+import nu.yona.server.crypto.seckey.StringFieldEncryptor;
 import nu.yona.server.util.TimeUtil;
 
 @Entity
 @Table(name = "USER_DEVICES")
 public class UserDevice extends DeviceBase
 {
+	enum VpnAccountType
+	{
+		LEGACY, STANDARD
+	}
+
 	@Type(type = "uuid-char")
 	@Column(name = "user_private_id")
 	private UUID userPrivateId;
 
+	@Convert(converter = StringFieldEncryptor.class)
```

Comment:
Why is it important to encrypt the VPN account type?

Reply:
It allows grouping users as old and new ones, which would be helpful when trying to break anonymity.

Reply:
OK, it adds a bit of clutter (converting enum to string, adding random bytes, removing random bytes when matching etc.) which is better to avoid. I think the risk is not that big.